apps/noaabuoy/noaa_buoy.star

```diff
@@ -0,0 +1,147 @@
+"""
+Applet: NOAA Buoy
+Summary: Display buoy wave data
+Description: Display swell data for user specified buoy. Find buoy_id's here : https://www.ndbc.noaa.gov/obs.shtml Buoy must have height,period,direction to display correctly
+Author: tavdog
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+load("encoding/json.star", "json")
+load("cache.star", "cache")
+
+def fetch_data(buoy_id):
+    url = "http://wildc.net/wind/noaa_buoy_api.pl?buoy_id=%s" % buoy_id
```

Comment:
A few questions about this URL:
- Do you own this domain?
- How is this perl script being hosted?
- Can we make it HTTPS? It looks like there is a certificate for the domain

I think we can support this so long as it is your domain and it's not on a server in your closet 😄 . If this app gets popular, we may also want to figure out a way to get this app onto a more standardized API.

Reply:
I do own this domain and it's hosted on dreamhost.com.  I'm all for making it more standardized.  This was just a quick scrape job to get the data out of noaa's database.